src/win/winpty.cc

```diff
@@ -62,11 +67,10 @@ static bool remove_pipe_handle(int handle) {
 }
 
 void throw_winpty_error(const char *generalMsg, winpty_error_ptr_t error_ptr) {
-  std::stringstream why;
+  std::wstringstream why;
   std::wstring msg(winpty_error_msg(error_ptr));
-  std::string msg_(msg.begin(), msg.end());
-  why << generalMsg << ": " << msg_;
-  Nan::ThrowError(why.str().c_str());
+  why << generalMsg << ": " << msg;
```

Comment:
Something I'm confused about is how wstringstream is able to consume non-wide `const char*` strings?

Reply:
I confirmed the following works:
```cpp
		std::wstringstream wss;
		const char* myCharStr = "world!";
		wss << "Hello " << myCharStr << std::endl;
		std::cout << std::string(from_wstring(wss.str().c_str()));
```